stable/wordpress/templates/ingress.yaml

```diff
@@ -1,39 +1,37 @@
 {{- if .Values.ingress.enabled }}
-{{- range .Values.ingress.hosts }}
 apiVersion: extensions/v1beta1
 kind: Ingress
 metadata:
-  name: "{{- printf "%s-%s" .name $.Release.Name | trunc 63 | trimSuffix "-" -}}"
+  name: {{ template "wordpress.fullname" . }}
   labels:
-    app: "{{ template "fullname" $ }}"
-    chart: "{{ template "wordpress.chart" $ }}"
+    app: "{{ template "wordpress.name" . }}"
+    chart: "{{ template "wordpress.chart" . }}"
     release: {{ .Release.Name | quote }}
     heritage: {{ .Release.Service | quote }}
```

Comment:
When I encountered this issue, I was able to fix it by updating these two labels as 

```yaml
    release: {{ $.Release.Name | quote }}
    heritage: {{ $.Release.Service | quote }}
```

Reply:
The reason to change the ingress configuration is using the same Ingress object with different rules per host defined instead of multiples ingress resources.